### ISSUE TITLE
Can't make sample on MacOS: error expected associated value for arg --configuration

### ISSUE THREAD

**Issue by virtualandy** [IBM-Swift/Kitura-Sample#32]
Hi! This looks like a ton of fun, just have a question about a problem I'm seeing trying to install/build on macOS.

I downloaded Xcode Beta and the associated toolchain (both 3.0-PREVIEW-6 and the development snapshot from Aug 23rd). Added it to my path (see below) but am getting this error when I do `make run`:

`error: expected associated value for argument: --configuration`

Here's the full error:
```bash
--- Running build on Darwin
--- Build scripts directory: Package-Builder/build
--- Checking swift version
swift --version
Apple Swift version 3.0-dev (LLVM 17c4c6cf7a, Clang 4ca9e01a7c, Swift fa36206531)
Target: x86_64-apple-macosx10.9
--- Checking swiftc version
swiftc --version
Apple Swift version 3.0-dev (LLVM 17c4c6cf7a, Clang 4ca9e01a7c, Swift fa36206531)
Target: x86_64-apple-macosx10.9
--- Checking git version
git --version
git version 2.9.3
--- Checking git revision and branch
git rev-parse HEAD
0b3c002b0ed5dd087e9b10278c7f6c0cfad8a955
git rev-parse --abbrev-ref HEAD
master
--- Invoking swift build
swift build -c
error: expected associated value for argument: --configuration
enter `/Library/Developer/Toolchains/swift-DEVELOPMENT-SNAPSHOT-2016-08-23-a.xctoolchain/usr/bin/swift-build --help' for usage information
make: *** [build] Error 1
```

Here's the contents of `/Library/Developer/Toolchains`:
```
drwxr-xr-x  7 root  wheel   238B Sep  2 11:58 swift-3.0-PREVIEW-6.xctoolchain
drwxr-xr-x  7 root  wheel   238B Sep  3 15:58 swift-DEVELOPMENT-SNAPSHOT-2016-08-23-a.xctoolchain
lrwxr-xr-x  1 root  wheel    81B Sep  3 15:58 swift-latest.xctoolchain -> /Library/Developer/Toolchains/swift-DEVELOPMENT-SNAPSHOT-2016-08-23-a.xctoolchain
```

And here's my PATH:
```bash
/Library/Developer/Toolchains/swift-latest.xctoolchain/usr/bin:/...<bunch of nvm and rvm junk>
```

Thanks for any help!

**Comment by vadimeisenbergibm** [comment]
@virtualandy There is a PR https://github.com/IBM-Swift/Package-Builder/pull/19 to fix that. Meanwhile, you can do the following - replace the `Makefile` in your `Package-Builder/build` directory with the `Makefile` from the PR - https://raw.githubusercontent.com/vadimeisenbergibm/Package-Builder/1b9404c2760c6eb213f866c12677e6600b7b0678/build/Makefile

**Comment by shmuelk** [comment]
@virtualandy You can also simply do a swift build to build it, followed by .build/debug/Kitura-Sample to run it.

**Comment by vadimeisenbergibm** [comment]
@virtualandy the issue was resolved by merging PR IBM-Swift/Package-Builder#19. Just remove `Package-Builder` directory and rerun `make run`

**Comment by virtualandy** [comment]
@vadimeisenbergibm worked perfectly after that PR closed and I removed the dir as you suggested. Thanks!

@shmuelk that also worked fine. My makefile debugging skills leave a lot to be desired, thanks for the simplified instructions too!
